TankDriver.App/Models/BulletSpaceModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using TankDriver.Logic;
+
+namespace TankDriver.Models
+{
+	internal class BulletSpaceModel: IModel
+	{
+		private BulletSpace _bulletSpace;
+		private TextureStorage _textureStorage;
+
+		public BulletSpaceModel (BulletSpace bulletSpace)
+		{
+			_bulletSpace = bulletSpace;
+		}
+
+		public void LoadBulletTextures(IModel bulletModel)
+		{
+			bulletModel.LoadTextures (_textureStorage);
+		}
+
+		#region IModel implementation
+
+		public IUnit GetUnit ()
+		{
+			return _bulletSpace;
+		}
+
+		public void LoadTextures (TextureStorage textureStorage)
+		{
+			_textureStorage = textureStorage;
+		}
+
+		public void Render (SpriteBatch spriteBatch)
+		{
+			foreach (var bullet in _bulletSpace.Bullets) {
+				bullet.GetModel ().Render (spriteBatch);
+			}
+		}
+
+		#endregion
+	}
+}
+
```

Comment:
> Will the enemies need their own EnemySpace then?

Sounds like a good idea for me.

> For now, let's say that your implementation (with `BulletSpace` and stuff) is okay, but I'll need to think about game engine design more later. I reserve the right to refactor this implementation.

Sure, refactor as you see needed. I can only propose stuff. :) What about `BulletSpaceModel`? Should I get rid of that and leave only `BulletSpace`?